build_scripts/build_usd.py

```diff
@@ -1024,6 +1024,41 @@ def InstallTBB_Windows(context, force, buildArgs):
         CopyFiles(context, "lib\\intel64\\vc14\\*.*", "lib")
         CopyDirectory(context, "include\\serial", "include\\serial")
         CopyDirectory(context, "include\\tbb", "include\\tbb")
+        CopyFiles(context, "cmake\\TBBConfig.cmake", "lib\\cmake\\TBB")
+        CopyFiles(context, "cmake\\TBBConfigVersion.cmake", "lib\\cmake\\TBB")
+
+        PatchFile(os.path.join(context.instDir, "lib\\cmake\\TBB\\TBBConfig.cmake"),
+                  [
+                      (
+                        "# TBBConfigVersion.cmake defines TBB_VERSION",
+                        "# TBBConfigVersion.cmake defines TBB_VERSION\n" +
+                        "#\n" +
+                        "# THIS FILE HAS BEEN MODIFIED FROM ITS ORIGINAL VERSION"      
+                      ),
+                      (
+                        "get_filename_component(_tbb_root \"${_tbb_root}\" PATH)",
+                        "get_filename_component(_tbb_root \"${_tbb_root}\" PATH)\n" +
+                        "get_filename_component(_tbb_root \"${_tbb_root}\" PATH)\n" +
+                        "get_filename_component(_tbb_root \"${_tbb_root}\" PATH)"
+                      ),
+                      (
+                        "get_filename_component(_tbb_lib_path \"${_tbb_root}/bin/${_tbb_arch_subdir}/${_tbb_compiler_subdir}\" ABSOLUTE)",
+                        "get_filename_component(_tbb_lib_path \"${_tbb_root}/bin\" ABSOLUTE)"
+                      ),
+                      (
+                        "${_tbb_root}/lib/${_tbb_arch_subdir}/${_tbb_compiler_subdir}/${_tbb_component}.lib",
+                        "${_tbb_root}/lib/${_tbb_component}.lib"
+                      ),
+                      (
+                        "${_tbb_root}/lib/${_tbb_arch_subdir}/${_tbb_compiler_subdir}/${_tbb_component}_debug.lib",
+                        "${_tbb_root}/lib/${_tbb_component}_debug.lib"
+                      ),
+                      (
+                        "if (EXISTS \"${_tbb_debug_lib}\")",
+                        "if (EXISTS \"${_tbb_debug_lib}\")\n" +
+                        "               set_property(TARGET TBB::${_tbb_component} APPEND PROPERTY INTERFACE_COMPILE_DEFINITIONS \"TBB_USE_DEBUG=1\")"
+                    )
+                  ])
```

Comment:
Do the copied CMake files (`cmake/TBBConfig.cmake` and `cmake/TBBConfigVersion.cmake`) also need to be removed on Windows similar to MacOS and Linux platforms?

Reply:
No, this isn't required on Windows - Windows uses a pre-built binary zip that straight copies things out to the target directories.  Linux / Mac build TBB from source, and when TBB builds it will put the `cmake` artifacts in the `cmake` folder, which are then copied to `lib/cmake/TBB` to be consistent with `MaterialX` and `OpenSubdiv`